### PR TITLE
Fix outdated fluentd image version

### PR DESCRIPTION
**Pull request by webwurst**
This PR:

- Fix outdated fluentd image version

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/efk-stack-app/values.yaml [hunk 1]
```diff
@@ -87,10 +87,6 @@ fluentd-elasticsearch:
       systemInputConf: false
   podSecurityPolicy:
     enabled: true
-  image:
-    registry: quay.io
-    repository: giantswarm/fluentd
-    tag: v3.0.0@sha256:2cc9334d646b14a002e71f0b0c34eca411e35f07c88f0bac9783805fb13c0190
```
Comment (ubergesundheit) [inline helm/efk-stack-app/values.yaml:93]: Could you explain why removing these values is enough? Seems now the value from https://github.com/giantswarm/efk-stack-app/blob/master/helm/efk-stack-app/charts/fluentd-elasticsearch/values.yaml#L7 are picked up. Is this correct?

Comment (webwurst) [reply helm/efk-stack-app/values.yaml:93]: Yes.

**Comment by webwurst** [comment]
While this should be fixed anyway, it seems not to fix the issue I wanted it to. Putting on hold for now.

**Comment by webwurst** [comment]
Fixed in https://github.com/giantswarm/efk-stack-app/pull/61

**Pull request closed without merging**